requirements/dev.txt

```diff
@@ -13,3 +13,6 @@ selenium==3.12.0
 djangorestframework==3.8.2
 geopy==1.15.0
 pinax-notifications==5.0.3
+geoip2==3.0.0
+django-ipware==3
+django-multiupload
```

Comment:
please pin a specific version

Reply:
Also you do not need add these to both of the files, because dev inherits from prod here, so just keep adding in prod, make sure you update for the other PR too.

Reply:
Sure!